apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -1,4175 +1,1000 @@
-obliged
-observation
-observations
-observe
-observed
-observers
 obtain
 obtained
-obvious
 obviously
-occasion
-occasional
-occasionally
-occasions
-occupation
-occupied
-occur
-occurred
-occurrence
-occurring
-occurs
-ocean
-oct
-october
-odd
 of
 off
-offer
-offered
-offering
-offers
-office
-officer
-officers
-offices
-official
-officials
 often
 oh
-oil
 ok
 okay
-okt
-old
-older
-oldest
+om
 on
 once
 one
 one's
 ones
 only
-onset
 onto
-op
-open
-opened
-opening
-openly
 opens
-operate
-operated
-operating
-operation
-operational
-operations
-operator
-opportunities
-opportunity
-opposed
-opposite
-opposition
-optimal
 or
-oral
-orchestra
 ord
-order
-ordered
-ordering
-orderly
-orders
-organ
-organic
-organization
-organizations
-organized
-origin
-original
-originally
+org
 other
 others
 otherwise
 ought
 oughtn't
+oughtnt
 our
 ours
+ours
+ourselves
 ourselves
 out
-outcome
-outdoor
-outlook
-output
-outside
-outstanding
 over
-over-all
 overall
-overcome
-overseas
-overwhelming
+owing
 own
-owned
-owner
-owners
-ownership
-oxidation
-oxygen
 p
-pace
-pack
-package
-packed
-page
-pages
-paid
-pain
-painful
-paint
-painted
-painter
-painting
-paintings
-pair
-pale
-panel
-panels
-panic
-paper
-papers
-parade
-paragraph
-parallel
-parent
-parents
-parked
-parking
-parks
+pa
 part
-part-time
 parted
-partially
-participate
-participation
-particle
-particles
 particular
 particularly
-parties
 parting
-partisan
-partly
-partner
 parts
-party
-pass
-passage
-passages
-passed
-passenger
-passengers
-passes
-passing
-passion
 past
-patent
-path
-pathology
-patience
-patient
-patients
-patrol
-pattern
-patterns
-pause
-paused
-pay
-paying
-payment
-payments
-pdt
-peace
-peaceful
-peas
-peculiar
-peered
-pencil
-penny
-people
-peoples
+pe
 per
-percent
-percentage
-perception
-perfect
-perfectly
-perform
-performance
-performances
-performed
 perhaps
-period
-periods
-permanent
-permission
-permit
-permits
-permitted
-person
-personal
-personality
-personally
-personnel
-persons
-perspective
-persuaded
-pertinent
-petitioner
-pfc
+pf
+pg
 ph
-ph.d
-phase
-phases
-phd
-phenomena
-phenomenon
-philosophical
-philosophy
-phone
-phrase
-physical
-physically
-physics
-piano
-pick
-picked
-picture
-pictures
-piece
-pieces
-pile
-pilot
-pink
-pioneer
-pipe
-pistol
-pitch
+pk
+pl
 place
 placed
 places
-placing
-plain
-plan
-plane
-planes
-planet
-planetary
-planets
-planned
-planning
-plans
-plant
-plants
-plaster
-plastic
-plastics
-plate
-plates
-platform
-play
-played
-player
-players
-playing
-plays
-pleasant
 please
-pleased
-pleasure
-plenty
-plot
-plug
-plural
 plus
 pm
-pocket
-poem
-poems
-poet
-poetic
-poetry
-poets
+pmid
+pn
 point
 pointed
 pointing
 points
-police
-policeman
-policies
-policy
-political
-politicians
-politics
-polynomial
-pond
-pool
-poor
-popular
-populate
-population
-porch
-port
-portion
-pose
-position
-positions
-positive
-possessed
-possession
-possibilities
-possibility
 possible
 possibly
-post
-posted
```

Comment:
We'll probably need to readd stuff like "posted", "published" etc. later on as those are common extraction artifacts ("This blog post was posted on May 1, 2019"). But let's keep it absent for now for the sake of a clean slate.